hooks/pre_validations/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,54 @@
+MONGO_ENGINE_COMPLETE = '/etc/foreman-installer/.mongo_engine_upgrade'.freeze
+MONGO_DIR = '/var/lib/mongodb/'
+
+# Pull MongoDB connection info from answer file
+if answers['katello']
+  MONGO_HOST = param('katello', 'pulp_db_seeds')
+else
+  MONGO_HOST = param('foreman_proxy_content', 'pulp_db_seeds')
+end
+
+def fail_and_exit(message)
+  Kafo::Helpers.log_and_say :error, message
+  kafo.class.exit 1
+end
+
+def disk_space
+  # Check diskspace in /tmp
+  logger.info 'Checking available diskspace in /var/tmp for upgrade'
+  total_space = `df -k --exclude-type=tmpfs /var/tmp`.split("\n")[-1].split[3].to_i
```

Comment:
You can leave out the exclude-type because you're directly asking for a location. You can also add `--output=avail` so it's the only column: `.split("\n").last.to_i` is less magic and easier to read.